src/components/Shared/Icons/AndroidIcon.tsx

```diff
@@ -3,15 +3,7 @@ import SvgIcon from '@mui/material/SvgIcon';
 function AndroidIcon() {
   return (
     <SvgIcon viewBox='0 0 30 30'>
-      <g id='SVGRepo_bgCarrier' strokeWidth='0'></g>
-      <g
-        id='SVGRepo_tracerCarrier'
-        strokeLinecap='round'
-        strokeLinejoin='round'
-      ></g>
-      <g id='SVGRepo_iconCarrier'>
-        {' '}
-        <title>android</title>{' '}
+      <g>
```

Comment:
even this `g` is not necessary. You can keep just the `path`.
(same below)

`g` means `group`, it is meant to group together several other elements to apply some common characteristics or use it as a reference later (if they have an `id`). No need for that here because we have just one path and we don't use it as a reference.